src/telegram.js

```diff
@@ -54,6 +54,12 @@ var TelegramBot = function (token, options) {
   if (options.webHook) {
     this._WebHook = new TelegramBotWebHook(token, options.webHook, this.processUpdate);
   }
+
+  // id generator for reply messages
+  var idx=0;
+  this.newReplyId = function() {
```

Comment:
This naming may bring name-conflicts in the future.

This would be preferable:

```js
let replyListenersIndex = 0;
this._nextReplyListenerId = function() { return replyListenersIndex++; };
```

We need the `_` in the name to signify it is **not** exported and should **not** be used by library consumers.